src/helpers/new-suggestion.helper.ts

```diff
@@ -14,6 +15,29 @@ export function isSongInformationInvalid<TFieldNames, TFieldName>(
   return watch("title") == "" || watch("artist").length == 0 || watch("motivation") == ""
 }
 
-export function isInstrumentSuggestionInvalid(instruments: NewSuggestionInstrument[]) {
+export function isInstrumentSuggestionInvalid(instruments: ISuggestionInstrument[]) {
   return instruments.length < 1
 }
+
+export const mapInstruments = (
+  suggestion: ISuggestion,
+  suggestionId: string
+): SongInstrumentDatabaseOperation[] => {
+  return suggestion.instruments.map(({ instrument, description }) => {
+    return { song_id: suggestionId, instrument_id: instrument.id, description: description }
+  })
```

Comment:
Could be written as
```ts
return suggestion.instruments.map(({ instrument, description }) => ({
    song_id: suggestionId,
    instrument_id: instrument.id,
    description: description,
  }))
  ```

Reply:
Added. Though it doesn't change the readability, performance, or any other factor of the code. does it matter?